src/app/functions/server/dashboard.ts

```diff
@@ -269,21 +358,58 @@ export function getDashboardSummary(
       }
     }
     /** c8 ignore stop */
+
+    if (b.isResolved) summary.dataBreachResolvedNum++;
   });
 
   // count unique breaches
   summary.dataBreachTotalNum = subscriberBreaches.length;
+  summary.dataBreachUnresolvedNum =
+    summary.dataBreachTotalNum - summary.dataBreachResolvedNum;
   const isBreachesOnly = summary.dataBrokerTotalNum === 0;
-  summary.sanitizedExposures = sanitizeExposures(
-    summary.allExposures,
-    summary.totalExposures,
+
+  // count unresolved exposures
+  summary.unresolvedExposures = Object.keys(summary.allExposures).reduce(
+    (a, k) => {
```

Comment:
Not entirely sure, but I believe you can save yourself a couple of type assertions below if you add type parameters for the function arguments:

```suggestion
    (a: Exposures, k: keyof Exposures) => {
```

Reply:
Tried that previously, and this is the error that occurs:
```
No overload matches this call.
  Overload 1 of 3, '(callbackfn: (previousValue: string, currentValue: string, currentIndex: number, array: string[]) => string, initialValue: string): string', gave the following error.
    Argument of type '(a: Exposures, k: keyof Exposures) => Exposures' is not assignable to parameter of type '(previousValue: string, currentValue: string, currentIndex: number, array: string[]) => string'.
      Types of parameters 'a' and 'previousValue' are incompatible.
        Type 'string' is not assignable to type 'Exposures'.
 ...
```